data/speakers.ts

```diff
@@ -456,7 +456,7 @@ const speakers: Speaker[] = [
       linkedin: 'https://www.linkedin.com/in/arunmichaeldsouza/',
     },
     talks: [],
```

Comment:
Hi @ArunMichaelDsouza , thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Not a problem! I've updated the talks list.

Reply:
Thank you! Everything is perfect now!

Reply:
Great!